libs/client/client.ts

```diff
@@ -1,16 +1,11 @@
 import axios from "axios";
 import type { Commitment } from "../nightfall/types";
-import path from "path";
-import { parentLogger } from "../utils";
+import { logger } from "../utils";
 import type { NightfallZkpKeys } from "../nightfall/types";
 import type { RecipientNightfallData } from "libs/transactions/types";
-import { NightfallSdkError } from "../utils/error";
+import { NightfallSdkError } from "../utils";
```

Comment:
```suggestion
import { logger, NightfallSdkError } from "../utils";
```

Or are you keeping them separate for readability (?)

On the other hand, if you accept the suggestion, please update the other `NightfallSdkError` imports :pray:

Reply:
Very good suggestion, I will correct it across the project. Thanks 🙏